eldritch/events.py

```diff
@@ -2423,9 +2442,11 @@ def resolve(self, state):
       return False
     if self.check.successes >= 1:
       self.evaded = True
+      self.character.avoid_monsters.append(self.monster)
       return True
     self.damage = Loss(
         self.character, {"stamina": self.monster.damage("combat", state, self.character)})
+    self.character.movement_points = 0
```

Comment:
Nitpick: I would either put this line above or below the use of self.damage so that you don't split the definition from the place where it's appended to the stack.

Reply:
This nitpick and others in events.py look like they got missed?